File: AssignReact/assignreact/src/EmployeeModel.ts
class Employee {
    empId: Number
    empName: string
    managerId: Number
    empSalary: Number

    constructor(empId: Number, empName: string, managerId: Number, empSalary: Number) {
        this.empId = empId;
        this.empName = empName;
        this.managerId = managerId;
        this.empSalary = empSalary;
    }
}

export default Employee;

File: AssignReact/AssignBackAPI/Models/EmployeeModel.cs
using System.ComponentModel.DataAnnotations;

namespace Assignment.Models
{
    public class EmployeeModel
    {
        [Key, Required]
        public int EmpId { get; set; }
        [Required]
        public string EmpName { get; set; }
        public int ManagerId { get; set; }
        [Required]
        public int EmpSalary { get; set; }

    }
}
